bundles/org.openhab.ui/web/src/pages/settings/things/channel/channel-general-settings.vue

```diff
@@ -6,12 +6,13 @@
                        info="Required. Note: cannot be changed after the creation"
                        required validate pattern="[A-Za-z0-9_\-]+" error-message="Required. A-Z,a-z,0-9,_,- only"
                        @input="channel.id = $event.target.value" />
-        <f7-list-item v-else :disabled="disabled" media-item class="channel-item" title="Channel UID">
-          <div slot="subtitle">
+        <f7-list-input v-else label="Channel UID" type="text" :input="false" disabled>
```

Comment:
This was set to list-item instead of list-input because channel UIDs tend to be long

Reply:
It now looks like this: 
Desktop:
![image](https://github.com/openhab/openhab-webui/assets/73423173/22be0bee-9669-458a-84bc-4d1a9ae40631)

iPhone:
![image](https://github.com/openhab/openhab-webui/assets/73423173/c76c6254-77d7-44b8-9b63-379a791e3e4b)

Reply:
I think I will revert to the old look here, but keep the transformation UID the new way (those usually aren't too long).